articles/azure/azs-how-create-S2S-VPN-azurestack-to-azure.md

```diff
@@ -60,23 +60,25 @@ First, you'll need to create a virtual network. This virtual network will be abl
 
 5. In the **Create virtual network** blade, enter the following information:
 
-   - **Name** - The name of the virtual network.
-
-   - **Address Space** - The virtual network's address range in CIDR notation.
-
    - **Subscription** - This is your UKCloud for Microsoft Azure subscription.
 
    - **Resource Group** - Select an existing resource group, or create a new one by typing a name for your new resource group.
 
-   - **Location** - This will be `frn00006`, which is the location of the Azure Stack Hub.
+   - **Name** - The name of the virtual network.
 
-   - **Subnet Name** - The name of the first subnet within the virtual network.
+   - **Region** - This will be `frn00006`, which is the location of the Azure Stack Hub.
 
-   - **Address Range** - The subnet's address range in CIDR notation (for example, 192.168.1.0). It must be contained by the address space of the virtual network. The address range of a subnet which is in use can't be edited.
+        ![Create new virtual network](images/azs-browser-create-virtual-network.png)
 
-     ![Create new virtual network](images/azs-browser-create-virtual-network.png)
+   - **Address Space** - The virtual network's address range in CIDR notation. This must not overlap the address space of the public Azure virtual network.
 
-6. Click **Create**.
+   - **Subnet Name** - The name of the first subnet within the virtual network.
+   
+   - **Subnet Address Range** - The subnet's address range in CIDR notation (for example, 192.168.1.0). It must be contained by the address space of the virtual network. The address range of a subnet which is in use can't be edited.
+
+     ![Create new virtual network2](images/azs-browser-create-virtual-network2.png)
```

Comment:
Image isn't showing. Maybe the case of png?

Reply:
Resolved by fixing file extension.